packages/accounts-controller/src/AccountsController.ts

```diff
@@ -101,18 +105,22 @@ export type AccountsControllerSelectedAccountChangeEvent = {
   payload: [InternalAccount];
 };
 
+export type AllowedAccountsControllerEvents =
+  | SnapStateChange
+  | KeyringControllerAccountRemovedEvent
```

Comment:
Do we really need this one? It's only required when we `.subscribe` to an event? And I couldn't find it?

Reply:
Its not currently used right now, but would be once there is the AccountAddedEvent from the keyring controller. I guess for now it would make sense to just remove it.